lib/stdlib/src/erl_stdlib_errors.erl

```diff
@@ -633,6 +635,11 @@ check_io_arguments([Type|TypeT], [Arg|ArgT], No) ->
              check_io_arguments(TypeT, ArgT, No+1)]
     end.
 
+format_json_error(_F, _As, #{position := Position}) ->
+    [{general, io_lib:format("Occured at position ~w", [Position])}];
```

Comment:
Is the position a character? or a byte?

Reply:
What is a character in unicode?
I wrote position for a reason :-)
 @michalmuskala